akka-docs/src/main/paradox/routing.md

```diff
@@ -755,13 +755,6 @@ continue to process its messages as normal, eventually processing the `PoisonPil
 cause the routee to stop. After all routees have stopped the router will itself be stopped
 automatically unless it is a dynamic router, e.g. using a resizer.
 
-@@@ note
-
-Brendan W McAdams' excellent blog post [Distributing Akka Workloads - And Shutting Down Afterwards](http://bytes.codes/2013/01/17/Distributing_Akka_Workloads_And_Shutting_Down_After/)
```

Comment:
We could link to https://web.archive.org/web/20200218184015/http://bytes.codes/2013/01/17/Distributing_Akka_Workloads_And_Shutting_Down_After/ - but perhaps it's not so relevant anymore nowadays.